docs/README.md

```diff
@@ -18,25 +29,23 @@ KubeArmor running on the edge node provides visibility and protection for all th
 
 **Enforcement:** KubeArmor can be used to apply security postures at the kernel level (using LSMs like AppArmor, BPF-LSM). It can protect both the host and workloads running on it by enforcing either some predefined security policies or automatically generated least permissive security policies (using Discovery Engine).
 
-KubeArmor already supports k8s-orchestrated workloads and provides [KVMService](https://github.com/kubearmor/kvm-service) that allows orchestrating security policies to VMs for non-k8s environments.
-With v0.5.5 release, KubeArmor now supports standalone un-orchestrated containers. KubeArmor in this mode supports both enforcement and observability of the host and the containers running on it.
+KubeArmor already supports k8s-orchestrated workloads and provides [KVMService ](https://github.com/kubearmor/kvm-service){:target="_blank"}{: .externalLink} that allows orchestrating security policies to VMs for non-k8s environments. With v0.5.5 release, KubeArmor now supports standalone un-orchestrated containers. KubeArmor in this mode supports both enforcement and observability of the host and the containers running on it.
 
 ## KubeArmor on Open Horizon
 
 > **Note**
 > This guide assumes both the Open Horizon Management Hub and Agent VM are running Ubuntu 20.04.
-We will first need to install Open Horizon Management Hub and Agent node components. For that please follow the [Open Horizon setup](https://github.com/kubearmor/KubeArmor/wiki/Open-Horizon-setup) guide.
-We also assume that [Open Horizon Home Assistant service](https://github.com/open-horizon-services/service-homeassistant) is running on the agent edge node.
+We will first need to install Open Horizon Management Hub and Agent node components. For that please follow the [Open Horizon setup ](https://github.com/kubearmor/KubeArmor/wiki/Open-Horizon-setup){:target="_blank"}{: .externalLink} guide. We also assume that [Open Horizon Home Assistant service ](https://github.com/open-horizon-services/service-homeassistant){:target="_blank"}{: .externalLink} is running on the agent edge node.
 
-<p align="center"><img src="./OH-detailed.png" width="768"></p>
+![KubeArmor Open Horizon details](./OH-detailed.png)
 
 Now we will run KubeArmor as a systemd process on the Open Horizon Agent VM
 
 ## Installation KubeArmor, kArmor, and Discovery Engine
 
 * **KubeArmor Installation:**
 
-1. Download the [latest release](https://github.com/kubearmor/KubeArmor/releases) of KubeArmor
+1. Download the [latest release ](https://github.com/kubearmor/KubeArmor/releases){:target="_blank"}{: .externalLink} of KubeArmor
```

Comment:
> [latest release ]

[latest release]

Reply:
The extra space is intentional. It will separate the linked text from the launch glyph.

Reply:
Oh! I see. Thanks for clarifying.